libp2p/Service.cpp

```diff
@@ -150,6 +150,17 @@ void Service::heartBeat()
     });
 }
 
+/// reset whitelist: stop session which is not in whitelist
+void Service::resetWhitelist()
+{
+    std::cout << "reset white list start" << std::endl;
```

Comment:
Just fake function here. Implementation will be requested at net PR

Reply:
If that, you can use 
```
// TODO: explanation
```

Reply:
Done. All logic has been push here now.